client/src/connection/itc/index.ts

```diff
@@ -515,6 +508,8 @@ export class ITCSession extends Session {
    */
   private fetchResults = async () => {
     if (!this._html5FileName) {
+      this._pollingForLogResults = false;
+      this._skipPageHeaders = false;
```

Comment:
nit: it feels a little risky to set to false here. For example some exception before entering here. Or some other route may be added in fututre.
How about set false from CodeRunner finally? Or, the `fetchLog` and `run` function accepts a `skipPageHeader` parameter, so that it only affect a specific run and not session level?

Reply:
> Or, the fetchLog and run function accepts a skipPageHeader parameter, so that it only affect a specific run and not session level?
I don't know why I didn't just do this. My thoughts were that I didn't want to muck with the session interface, but this seems like the most reasonable approach.

Reply:
Updated in https://github.com/sassoftware/vscode-sas-extension/pull/880/commits/bfce3044abf0021f961b85cd12cc21406209e729